aztk/spark/client/job/helpers/submit.py

```diff
@@ -0,0 +1,111 @@
+import azure.batch.models as batch_models
+import azure.batch.models.batch_error as batch_error
+import yaml
+
+from aztk import error
+from aztk.internal.cluster_data import NodeData
+from aztk.spark import models
+from aztk.utils import helpers
+from aztk.utils.command_builder import CommandBuilder
+
+
+def __app_cmd():
+    docker_exec = CommandBuilder("sudo docker exec")
+    docker_exec.add_argument("-i")
+    docker_exec.add_option("-e", "AZ_BATCH_TASK_WORKING_DIR=$AZ_BATCH_TASK_WORKING_DIR")
```

Comment:
i think you can just do `-e MY_ENV` and docker will do the same

Reply:
Yeah -- this code has just been reshuffled, not updated. That is an easy refactor though.

Reply:
Actually, it looks like changing the `-e env=$env` to `-e env` does not work here. Possibly that syntax only works for the `docker run`, not `docker exec`. Not sure, but for now I'll just leave it as it works as is.